src/source-updater.js

```diff
@@ -272,38 +272,18 @@ export default class SourceUpdater extends videojs.EventTarget {
    * @param {Function} done the function to call when done
    * @see http://www.w3.org/TR/media-source/#widl-SourceBuffer-appendBuffer-void-ArrayBuffer-data
    */
-  appendBuffer({segmentInfo, type, bytes, videoSegmentTimingInfoCallback}, doneFn) {
+  appendBuffer(type, bytes, doneFn) {
```

Comment:
I changed this from `(type, bytes, doneFn)` -> `({type, bytes, videoSegmentTimingInfoCallback}, doneFn)` when we needed to pass `videoSegmentTimingInfoCallback`. Now that we no longer need to, I put it back to this.

Reply:
Any reason why not to keep the options object method signature? I would like to see `segmentInfo` continue to be pass through here to the appendBuffer action.

Reply:
Yeah I will put it back to how it was